src/html/index.html

```diff
@@ -20,20 +20,127 @@
       <p>
         <button type="button" class="button" data-open-modal="success">modal--success link</button>
       </p>
+      <section class="main__banner banner">
+        <h1 class="visually-hidden">Информация про Ясли</h1>
+        <h2 class="visually-hidden">Главный баннер</h2>
+        <p class="banner__header">Идеальные ясли для ваших малышей</p>
+        <p class="banner__description">для детей от 1,6</p>
+        <button type="button" class="main__button">
+          <span>Свяжитесь с нами</span>
+        </button>
+      </section>
+      <aside class="main__info info-box">
```

Comment:
```suggestion
      <section class="main__info info-box">
```

Почему это вдруг aside непонятно...
Это явно не информация, которая не относится к содержимому страницы

Reply:
если целостность страницы не меняется при удалении этого блока,то можно выделить его в aside, разве нет?
но я без проблем поменяю на section

Reply:
Это только часть характеристики. Помимо этого, aside должен быть дополнительной, вспомогательной информацией.
Мой любимый пример — заметка на полях. То есть, мы например написали статью про научную фантастику. И там мы упоминаем Азимова и Бредбери. В статье мы просто их упоминаем, но делаем выноску, где приводим годы жизни, самые известные произведения и прочее. Имеет это отношение к статье? Нет. Статья может без этого обойтись? Да. Это просто доп информация, которая имеет косвенное отношение к содержанию